universe/Condition.h

```diff
@@ -93,6 +93,9 @@ struct FO_COMMON_API Condition {
     virtual unsigned int GetCheckSum() const
     { return 0; }
 
+    //! Makes a clone of value into new owning pointer. Required for Boost.Python which
```

Comment:
Makes a clone of this Condition in a new owning pointer. Required for Boost.Python, which doesn't support move semantics for returned values.

Reply:
Fixed.